tests/tests_twobody/test_orbit.py

```diff
@@ -1140,3 +1140,9 @@ def test_can_set_iss_attractor_to_earth():
     )
     iss = iss.change_attractor(Earth)
     assert iss.attractor == Earth
+
+
+def test_issue_916():
```

Comment:
Should we mark this as a remote test? Or, alternatively (better), mock the response.